src/integration/utils/fetchDom.js

```diff
@@ -31,10 +31,11 @@ const faultTolerantDomFetch = ({ url, runScripts, headers }) =>
       try {
         const response = await fetch(url, headers && { headers });
         const html = await response.text();
-        const dom = new JSDOM(
-          html,
-          runScripts ? { runScripts: 'dangerously' } : {},
-        );
+
+        const dom = new JSDOM(html, {
```

Comment:
In fact, can we also make this change on our branch too, as it stops the error message appearing in the console